moveit_ros/planning_interface/common_planning_interface_objects/src/common_objects.cpp

```diff
@@ -105,15 +106,17 @@ std::shared_ptr<tf2_ros::Buffer> getSharedTF()
   std::shared_ptr<tf2_ros::Buffer> buffer = s.tf_buffer_.lock();
   if (!buffer)
   {
-    tf2_ros::Buffer* raw = new tf2_ros::Buffer();
+    s.clock_ = std::make_shared<rclcpp::Clock>();
```

Comment:
You may want to explicitly define the clock type with RCL_ROS_TIME

Reply:
Done